config/v1/types_cluster_version.go

```diff
@@ -199,6 +199,21 @@ type ClusterVersionStatus struct {
 	// +listType=atomic
 	// +optional
 	ConditionalUpdates []ConditionalUpdate `json:"conditionalUpdates,omitempty"`
+
+	// relatedObjects is a list of objects that are "interesting" or related to this operator.
+	// `oc adm inspect` honors this field in any type to navigate and collect related data.
+	// Common uses are:
+	// 1. operator namespaces
+	// 2. operand namespaces
```

Comment:
We obviously want to include the `openshift-cluster-version` namespace for our operator, but "operand" gets a bit sticky with the CVO.  Do we do just our own namespace on the thin end, and build out CVO logging as needed to help debug things where we don't capture enough to debug a resource we're failing to reconcile?  Or [all ~1k resources release-manifests are asking us to manage][1] on the thick end?  Or pick some middle ground like "asking for all ClusterOperators is easy, and likely covers reporting on _most_ of the resources we care about"?  Or...?

[1]: https://github.com/openshift/enhancements/blob/6728b373c4aae7ad278ba940969bc9a8d28a0e34/dev-guide/cluster-version-operator/user/reconciliation.md#manifest-graph